src/retry.ts

```diff
@@ -89,6 +89,12 @@ export async function getRetryConfig(err: GaxiosError) {
 function shouldRetryRequest(err: GaxiosError) {
   const config = getConfig(err);
 
+  // If the user has explicitly aborted the request, don't retry.
+  // "type" is populated on the error objects created by node-fetch.
+  if (err.type === 'aborted') {
```

Comment:
So node-fetch is putting the “aborted” on... I guess a small concern would be that they change that at some point. Is it an advertised behavior that you think their versioning would guarantee?

Reply:
in their docs, it looks like they advocate using err.name, I've updated accordingly:

https://github.com/bitinn/node-fetch#request-cancellation-with-abortsignal